client/tailscale/tailscale.go

```diff
@@ -38,24 +38,59 @@ import (
 	"tailscale.com/tailcfg"
 )
 
-var (
-	// TailscaledSocket is the tailscaled Unix socket. It's used by the TailscaledDialer.
-	TailscaledSocket = paths.DefaultTailscaledSocket()
+// Local is the default LocalClient when using the legacy package-level functions.
+var Local LocalClient
```

Comment:
Maybe if this was not public it would be less of an issue? Seems like we want people to move away from it being a singleton, no need to expose it.

Reply:
removed.